cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -58,6 +58,7 @@ def get_data(self, _options, re_id):
 
 class FeatureEffectsView(ExperimentAccessMixin, TemplateJsonView):
     json_renderer = feature_reg_effects
+    # tsv_renderer = feature_reg_effects_tsv
```

Comment:
You didn't add TSV downloads to these tables, you don't need to modify this view at all.

Reply:
Yes, this I didn't touch this at all. Are you saying you want me to remove this commented out section? This may be carried over from the previous branch.

Reply:
Undo all the changes to this file